assets/styles/style.css

```diff
@@ -0,0 +1,416 @@
+}
+
+.comp-cont {
+  width: 100%;
+  margin: 25px auto;
+}
+
+.orange {
+  background-color: #f44120;
+}
+
+.orange h4 {
+  font-size: 18px;
+  padding: 0.8em 1em 0.8em 4.5em;
+  color: white;
+}
+
+.lorange {
+  background-color: #ffebe7;
+  font-size: 18px;
+  padding: 0.5em 1em 0.8em 4.5em;
+}
+
+.lorange h5 {
+  font-size: 11px;
+}
+
+.company-logo {
+  position: absolute;
+  width: 60px;
+  top: 12px;
+  left: 12px;
+}
+
+.pale {
+  background-color: #fffaf8;
+}
+
+.tags {
+  font-size: 15px;
+  line-height: 2em;
+  padding: 15px;
+}
+
+.bottom-sec {
+  border-top: 1px solid #f44120;
+  padding: 15px;
+  font-size: 13px;
+}
+
+.light-grey {
+  color: #aaa;
+}
+
+.cat-articles {
+  padding-top: 1.3em;
+}
+
+.info-art {
+  display: flex;
+  flex-direction: row;
+}
+
+.category-cont {
+  background-color: #fff4f2;
+}
+
+.category-info {
+  padding: 1em 1em;
+  font-size: 14px;
+}
+
+.category-cont img {
+  float: right;
+  width: 96px;
+  height: 96px;
+}
+
+.category-list ul {
+  padding-left: 1em;
+}
+
+.category-list li::before {
+  content: "\2022";
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+}
+
+.deals-header {
+  justify-content: space-between;
+  align-items: baseline;
+  margin: 1.5em 0;
+}
+
+footer {
+  color: #aaa;
+  font-size: 13px;
+}
+
+.social-icons {
+  margin-bottom: 25px;
+}
+
+.ftr-social {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: #131313;
+  height: 161px;
+  text-align: center;
+}
+
+.contact-us {
+  width: 70%;
+}
+
+.ftr-copyright {
+  height: 122px;
+  background-color: black;
+  text-align: center;
+  padding: 32.5px 0;
+}
+
+.footer-logo-img {
+  width: 37px;
+  height: 9px;
+}
+
+.stackpath {
+  width: 65px;
+  height: 9px;
+}
+
+.cross {
+  text-decoration: line-through;
+}
+
+.hf-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/hf.jpg);
+}
+
+.apps-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/n3.png);
+}
+
+.gear-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/l8.jpg);
+}
+
+.tech-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/t.png);
+}
+
+.creative-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/c.jpg);
+}
+
+.podium-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/p.png);
+}
+
+.insights-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/hf.jpg);
+}
+
+.launch-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/la.png);
+}
+
+.distract-img {
+  background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4)), url(../images/d.png);
+}
```

Comment:
If you notice there is a lot of repeated `linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4))`
One good way to overcome this is to use [CSS Variables](https://www.w3schools.com/css/css3_variables.asp). Example:

```
:root {
  --gradient: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4));
}

/* use it like this */
.hf-img {
  background-image: var(--gradient), url(../images/hf.jpg);
}
```
Now, it becomes very easy to make one change to affect the rest.